lib/entice/logic/npc.ex

```diff
@@ -21,13 +21,12 @@ defmodule Entice.Logic.Npc do
     :ok
   end
 
-
   def register(entity, map, name \\ "Dhuum", npc \\ %Npc{}) do
     entity |> Entity.attribute_transaction(fn (attrs) ->
       attrs
       |> Map.put(Name, %Name{name: name})
       |> Map.put(Position, %Position{pos: map.spawn})
-      |> Map.put(MapInstance, %MapInstance{map: map})
+      |> Map.put(MapInstance, %MapInstance{map: map})#TODO: remove?
```

Comment:
yes, and everywhere else where this is still used. we can integrate it back into the entities if we find that we need it